### PR TITLE
fix client path and node_name

### PR DESCRIPTION
**Pull request by alexpop**
Fixes issue #38 by:
 * using client key path from attributes
 * using trusted_certs path from attributes
 * using attribute for verify_api_cert
 * defaulting key and trusted_cert back to /etc/chef
 * using Chef::Config[:node_name] for node_name. The assumption that node_name is the same as the hostname is incorrect. Nodes can be bootstrapped in Chef with a node name that's different from the machine hostname

### CODE REVIEW

File: templates/default/push-jobs-client.rb.erb [hunk 1]
```diff
@@ -8,13 +8,11 @@
...
-chef_server_url  "<%= @chef_server_url %>"
-
-client_key                  '/etc/chef/secure/client.pem'
-trusted_certs_dir           '/etc/chef/secure/trusted_certs'
-verify_api_cert false
+chef_server_url   '<%= @chef_server_url %>'
+node_name         '<%= @node_name %>'
+client_key        '<%= @client_key_path %>'
+trusted_certs_dir '<%= @trusted_certs_path %>'
+verify_api_cert   false
```
Comment (manderson26) [inline templates/default/push-jobs-client.rb.erb]: I see there's a 'verify_api_cert' passed in the template definition; did you want to have <%= verify_api_cert %> here?

**Comment by manderson26** [comment]
:+1: looks good to me...

File: templates/default/push-jobs-client.rb.erb (hunk 1 above)
Comment (alexpop) [inline templates/default/push-jobs-client.rb.erb]: You are right, fixed and pushed. Thanks for the review

**Comment by alexpop** [comment]
Merged in version 2.4.1

**Pull request closed without merging**